.clj-kondo/hooks/metabase/models/setting.clj

```diff
@@ -0,0 +1,44 @@
+(ns hooks.metabase.models.setting
+  (:require [clj-kondo.hooks-api :as hooks]))
+
+(defn defsetting
+  "Rewrite a [[metabase.models.defsetting]] form like
+
+    (defsetting my-setting \"Description\" :type :boolean)
+
+  as
+
+    (let [_args [\"Description\" :type :boolean]}
```

Comment:
from reading the code, isn't this more like `(let [_ "Description" _ :type _ :boolean] (defn my-setting []) (defn my-setting! [_value-or-nil]))`

Reply:
Oh yeah you're right.